packages/pwa/app/commerce-api/auth.js

```diff
@@ -176,7 +176,8 @@ class Auth {
             }
             return this[authorizationMethod](credentials)
                 .catch((error) => {
-                    if (retries === 0 && error.message === 'EXPIRED_TOKEN') {
+                    const retryErrors = ['invalid refresh_token', 'EXPIRED_TOKEN']
```

Comment:
Since this is a constant, wanna declare it at the top of the file?

Reply:
I agree.. might be able to clean things up a little bit but defining error types: 

```
const EXPIRED_TOKEN = 'EXPIRED_TOKEN`
const INVALID_TOKEN = 'invalid refresh_token`

...

const retryErrors = [INVALID_TOKEN, EXPIRED_TOKEN]

if (retries === 0 && retryErrors.includes(error.message)) {
```

Something like that I guess.